include/rocksdb/advanced_options.h

```diff
@@ -483,6 +483,17 @@ struct AdvancedColumnFamilyOptions {
   // Default: false
   bool level_compaction_dynamic_level_bytes = false;
 
+  // The ratio to calcuate the total tolerant bytes of ingested files for one
+  // level.
+  // tolerant_bytes = ingest_tolerant_ratio /
+  //                  (level - base_level + 1) * level_max_bytes.
+  // With ingest tolerant bytes, the compaction wouldn't be triggered until
+  // bytes_without_ingest + ingest_bytes > level_max_bytes + tolerant_bytes.
+  // Only effective when level_compaction_dynamic_level_bytes is enabled.
+  //
+  // Default: 0
+  size_t ingest_tolerant_ratio = 0;
```

Comment:
Add a test in options/options_test.cc to test changing the value via `SetOptions`?

Reply:
It's already covered by adding https://github.com/tikv/rocksdb/pull/196/files#diff-9621ba7ca77a40881b657e843d9ce782c261c8030c518200f9e1cad358814d6fR2038

Reply:
that only check the value is changeable, but not the functionality.